README.md

```diff
@@ -1,40 +1,37 @@
-# Signal Android 
+# Signal Android Websockets-Only
 
 Signal is a messaging app for simple private communication with friends.
 
 Signal uses your phone's data connection (WiFi/3G/4G) to communicate securely, optionally supports plain SMS/MMS to function as a unified messenger, and can also encrypt the stored messages on your phone.
 
-Currently available on the Play store.
-
-*[![Play Store Badge](https://developer.android.com/images/brand/en_app_rgb_wo_60.png)](https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms)*
-
 #WebSocket Support#
 
 This branch adds rudimentary WebSocket-ONLY support to Signal Private Messenger.
```

Comment:
Nit: You could remove the ``-ONLY`` if we merge https://github.com/JavaJens/TextSecure/pull/40.